Content.Server/Cloning/CloningConsoleSystem.cs

```diff
@@ -148,49 +151,52 @@ public void UpdateUserInterface(EntityUid consoleUid, CloningConsoleComponent co
             _uiSystem.SetUiState(ui, newState);
         }
 
-        public void TryClone(EntityUid uid, EntityUid cloningPodUid, EntityUid scannerUid, CloningPodComponent? cloningPod = null, MedicalScannerComponent? scannerComp = null, CloningConsoleComponent? consoleComponent = null)
+        public void TryClone(EntityUid uid, EntityUid cloningPodUid, EntityUid scannerUid, CloningPodComponent cloningPod, MedicalScannerComponent? scannerComp = null, CloningConsoleComponent? consoleComponent = null)
         {
-            if (!Resolve(uid, ref consoleComponent) || !Resolve(cloningPodUid, ref cloningPod) || !Resolve(scannerUid, ref scannerComp))
-                return;
-
-            if (!Transform(cloningPodUid).Anchored || !Transform(scannerUid).Anchored)
-                return;
-
-            if (!consoleComponent.CloningPodInRange || !consoleComponent.GeneticScannerInRange)
+            if (!Resolve(uid, ref consoleComponent)
+                || !Resolve(scannerUid, ref scannerComp)
```

Comment:
I believe you can combine multiple Components into a single call to `Resolve`

Reply:
Thanks for the tip! Will do. :)